src/styles/utiles/_extends.scss

```diff
@@ -0,0 +1,3 @@
+%hover-duration {
+  transition-duration: $transition-duration;
```

Comment:
instead just transition-duration, use transition short-hand property and add property which transitions applies to

for example `transition: color 300ms ease-out` or `transition: transform 300ms ease-out`

https://developer.mozilla.org/en-US/docs/Web/CSS/transition